hbase-diagnostics/src/main/java/org/apache/hadoop/hbase/wal/WALPerformanceEvaluation.java

```diff
@@ -77,7 +78,7 @@
  * This class runs performance benchmarks for {@link WAL}. See usage for this tool by running:
  * <code>$ hbase org.apache.hadoop.hbase.wal.WALPerformanceEvaluation -h</code>
  */
-@InterfaceAudience.Private
+@InterfaceAudience.LimitedPrivate(HBaseInterfaceAudience.TOOLS)
```

Comment:
This was missed in previous commits as somehow this tool was in hbase-server and also didnot have HBaseInterfaceAudience.TOOLS which is the filter i was using to find tools needing refactor